doc/tech/README.md

```diff
@@ -130,32 +145,31 @@ The CNTT work is not done in a vacuum.  The intention from the beginning was to
 - OSM (ETSi Open Source MANO project)
 - VMware (While not an Open Source project, VMware is a commonly used platform used for VNF deployments in the telecom industry)
 
+<a name="3.4.1"></a>
+### 3.4.1 Relationship to ETSI-NFV
+
 The ETSI NFV ISG is very closely related to the CNTT, in that it is a group that is working on supporting technologies for NFV applications. To facilitate more collaboration as the project matures, the CNTT scope (**Figure 2** above) purposely references certain ETSI NFV reference points, as specified by [ETSI GS NFV 002](https://www.etsi.org/deliver/etsi_gs/NFV/001_099/002/01.02.01_60/gs_NFV002v010201p.pdf).
 
-The CNTT is also closely aligned with OVP, an open source, community-led compliance and verification program that demonstrates the readiness and availability of commercial NFV products and services, including cloud infrastructure and VNFs, using OPNFV. OVP combines open source-based automated compliance and verification testing for multiple parts of the NFV stack specifications established by ONAP, multiple SDOs such as ETSI and GSMA, and the LF Networking End User Advisory Group (EUAG).
+<p align="center"><img src="./figures/tech_relation_etsi.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 4:</b> CNTT Relationship with ETSI NFV.</p>
```

Comment:
The text says "CNTT Relationship with ETSI NFV" but the actual picture describes only the scope of ETSI NFV. Is this the right picture?

Reply:
@hdamker fixed in https://github.com/cntt-n/CNTT/pull/1433/commits/36ad1c9177358d096acab0a711ec5a0781cd1057